src/main/java/org/broadinstitute/hellbender/tools/BwaMemIndexImageCreator.java

```diff
@@ -20,24 +24,22 @@
 public final class BwaMemIndexImageCreator extends CommandLineProgram {
 
     @Argument(fullName = StandardArgumentDefinitions.INPUT_LONG_NAME,
-              shortName = StandardArgumentDefinitions.INPUT_SHORT_NAME,
-              doc = "Input reference fasta file. The five bwa index files are assumed living in the same directory with the same prefix.")
+            shortName = StandardArgumentDefinitions.INPUT_SHORT_NAME,
+            doc = "Input reference fasta file. The five bwa index files are assumed living in the same directory with the same prefix.")
```

Comment:
This doc is out of date, I think.  Index files aren't assumed to be living anywhere, or even to exist.